src/Dijkstra/AST/Either.agda

```diff
@@ -5,7 +5,7 @@
 -}
 
 -- As we are interested in using Either for error handling (and providing the Either-bail command
--- for this purpose), we call the Left type "Err"
+-- for this purpose), we call the left type "Err"
```

Comment:
`Haskell.Modules.Either` still has patterns for lowercase `left` and `right`.

It seems "Haskell" should be Haskell and only have upper case `Left` and `Right'

And Dijkstra should either use Agda inj1/2 (or lower case `left` and `right`)

Reply:
Good catch, thanks.  FIXED.